Sprint-1/implement/dedupe.test.js

```diff
@@ -7,16 +8,49 @@ Dedupe Array
 In this kata, you will need to deduplicate the elements of an array
 
 E.g. dedupe(['a','a','a','b','b','c']) target output: ['a','b','c']
-E.g. dedupe([5, 1, 1, 2, 3, 2, 5, 8]) target output: [5, 1, 2, 3, 8]
+
 E.g. dedupe([1, 2, 1]) target output: [1, 2]
+
 */
+// let input = [5, 1, 1, 2, 3, 2, 5, 8];
+// let target = [5, 1, 2, 3, 8]
+
+
+describe("dedupe", () => {
+
+
+    test("returns an empty array with empty array given", () => {
+        expect(dedupe([])).toEqual([]);
+    });
+    
+    test("returns the an unrepeated character of an array element with given array", () => {
+        expect(dedupe([5, 1, 1, 2, 3, 2, 5, 8])).toEqual([5, 1, 2, 3, 8]);
+    });
+    test("returns the original array element with given array", () => {
+        expect(dedupe([1,2,3,4])).toEqual([1,2,3,4]);
+    });
+    test("returns the an unrepeated character of an array element with a string and number array", () => {
+        expect(dedupe([1,'g',7,2,'r','g',3,4,'q'])).toEqual([1,'g',7 ,2 ,'r',3,4,'q']);
```

Comment:
#### Todo
Can you try using ChatGPT to make the description of these tests more accurate?